index.rst

```diff
@@ -165,49 +166,50 @@ Branch, Merge
 
 Any git repository that needs to be modified for additional ``rc`` releases
 should be **branched** and have the necessary changes merged to a release
-branch.  Eg., if changes were needed in ``v42.0.0.rc1`` a "release branch"
-along the lines of ``v42.0.x`` should be created in the repos that need changes.
+branch.  Eg., if changes were needed in ``v888.0.0.rc1`` a "release branch"
+along the lines of ``v888.0.x`` should be created in the repos that need
+changes.
 
 (**TBD**: merge to master and cherry-pick to release branch or merge to release
```

Comment:
How a change is cherry-pick from one branch to another, or an entire ticket branch, already merged to master, is "ported" to a Release Branch, shall be defined in developer guide, since this is a developer activity. 
Who approves the "porting" is a different topic, before it is done, it is a different topic, and probably will be the DMCCB.